wdl-lint/tests/lints/preamble-ws/source.errors

```diff
@@ -4,6 +4,12 @@ note[PreambleFormatting]: unnecessary whitespace in document preamble
 1 │     #@ except: DescriptionMissing
   │ ^^^^
 
+warning[MalformedLintDirective]: lint directive must be on its own line
```

Comment:
Should we correct this in the `preamble-ws` test's source so that we don't need to update the test's diagnostics baseline?

Reply:
That space is important to be checked for the `preamble-ws` rule, so we can't get rid of it. We could instead `except` `MalformedLintDirective` so this doesn't show up, but IMO we should leave it as an example of the bug. It should be a pretty trivial fix, and I can hopefully get to it today before we release, so don't think it matters much practically.